integration_tests/toMatchSnapshot/__tests__/basic-support.test.js

```diff
@@ -0,0 +1 @@
+test('snapshots', () => expect({apple: "updated value"}).toMatchSnapshot());
```

Comment:
Aha, I believe these files got added because I interrupted a test run, and it creates/deletes files during tests but when you interrupt it doesn't clean up properly. Mystery solved 👌